### PR TITLE
Feature/recolor

### PR DESCRIPTION
**Pull request by nerik**

### CODE REVIEW

**Comment by nerik** [comment]
screen shot 2016-10-18 at 12 47 00

File: styles/_settings.scss [hunk 1]
```diff
@@ -27,9 +27,9 @@ $cloudy-blue: #aadad1;
 $light-grey-blue: #8db6ca;
 $warm-pink: #f65e6e;
 
-$link-green: #bcda59;
-$link-blue: #2f7fab;
-$link-pink: $warm-pink;
+$qual-soybeans: #bcda59;
+$qual-soybeancake: #2f7fab;
+$qual-soybeanoil: $warm-pink;
```
Comment (agnlez) [inline styles/_settings.scss]: Why do you create another colour variable when you already have the `$warm-pink` one?

File: styles/_settings.scss [hunk 2]
```diff
@@ -27,6 +27,10 @@ $cloudy-blue: #aadad1;
 $light-grey-blue: #8db6ca;
 $warm-pink: #f65e6e;
 
+$link-green: #bcda59;
+$link-blue: #2f7fab;
+$link-pink: $warm-pink;
```
Comment (agnlez) [inline styles/_settings.scss]: Same as here: https://github.com/Vizzuality/TRASE-frontend/pull/42/commits/e6e2ccdec93d6074619fd0005bf9793da5ac53eb#r84235510

**Comment by agnlez** [comment]
Just a couple of comments, and make this PR mergeable to `develop`. Check them out and merge. Good job!

File: styles/_settings.scss (hunk 1 above)
Comment (nerik) [reply styles/_settings.scss]: - I want explicit semantic aliases for colors used in this recolor feature (they are used in 3 different places)
- `$qual-soybeanoil: $warm-pink;` is temporary anyways, as this Commodity recoloring will not make it to the final product (it's just used as a test in the DB)

**Pull request merged**